### DISCUSSION TITLE
The requested page could not be loaded.

### DISCUSSION THREAD

**Discussion by manapshymyr-OB** [developmentseed/eoAPI#179] [Q&A]
Hello all,

I tried to run with docker compose and everything is running. I could dump catalogs and items. While I open the stac browser, it shows me `This issue may occur when servers don't allow external access via web browsers (e.g., when CORS headers are not present). Please contact the service operator to resolve this issue.` How can I fix this? I tried to change to put `allow_origins=["*"],` in the `eoAPI/runtime/eoapi/stac/eoapi/stac/app.py` but did not help.

**Comment by zacdezgeo** [comment]
Hey @manapshymyr-OB,

Thank you for reaching out and detailing the issue with the STAC browser within eoAPI. Please provide us with additional information so we can better understand what might be causing the problem and guide you more effectively.

- **Configuration and Setup:** Please share the specific changes you made to the `app.py` file, especially regarding the `allow_origins=["*"]` setting, to ensure the modification was applied correctly.

- **Environment Details:** What version of eoAPI are you currently using? If you're working with a version from the `main` branch, please specify the commit hash or the date you pulled the changes. Also, could you let us know your Docker and Docker Compose versions, as this can help identify if the issue is related to a specific version?

- **Error Logs and Console Output:** Are there any error messages in the eoAPI logs or the console output of your web browser’s developer tools (especially in the Console and Network tabs) when you try to access the STAC browser? This information can be very revealing.

- **Browser Information:** Which web browser and version are you using? Browser-specific security settings can sometimes cause issues like the one you're experiencing.

- **Testing and Reproduction Steps:** Could you walk us through the steps you took from cloning the repository to encountering the error? This will help us understand your process and identify potential missteps or overlooked configurations.

Your detailed feedback on these points will help us improve the experience for you and others.

I'm looking forward to hearing back from you!

**Comment by emileten** [comment]
Thanks for the reporting your issue ! I can confirm on my end that both the following work, based on this commit, 9d6ac743c7a160baf7547889166d49d7a3581604 :

- the docker-compose application using the default runtimes, (`docker-compose.yml` file)
- the docker-compose application using the custom runtimes, (`docker-compose.custom.yml`)

Therefore, seconding @zacharyDez, it would be great if you could share more details about the steps you followed and your initial environment. In addition to what he mentioned, a screen of your browser page that shows the mentioned error would be useful. It sounds like an error on the STAC API side, not the browser side, as this is the kind of message I've seen in the past when I was trying to use the STAC browser with a disfunctional STAC API. But I would be more confident with more details !

Thank you!
